.github/workflows/validate.yml

```diff
@@ -14,3 +14,15 @@ jobs:
       uses: actions/checkout@v2
     - name: Validate
       uses: ./.github/actions/validate
+      env:
+        USERNAME: ${{ secrets.USERNAME }}
+        PASSWORD: ${{ secrets.PASSWORD }}
+        ACCOUNT_ID: ${{ secrets.ACCOUNT_ID }}
+        VOICE_APPLICATION_ID: ${{ secrets.VOICE_APPLICATION_ID }}
+        MESSAGING_APPLICATION_ID: ${{ secrets.MESSAGING_APPLICATION_ID }}
+        CALLBACK_URL: ${{ secrets.CALLBACK_URL }}
```

Comment:
What's being returned from `CALLBACK_URL`? It looks like this is only used for voice callbacks - maybe we should rename to `VOICE_CALLBACK_URL` to be explicit, and for my knowledge what BXML is being returned from this to keep the call alive?

Reply:
Yeah good callout. It is just for voice so renaming it would make more sense. No BXML is being returned because this is just checking to make sure that the call is created